deploy/helm/alerting/values.yaml

```diff
@@ -1,6 +1,6 @@
 image:
-  repository: quay.io/ecosystem-appeng/aiobs-metrics-alerting
-  tag: 0.1.5
+  repository: "overridden-by-makefile"
```

Comment:
Helm files now contain a static text indicating their values will be overridden by Makefile. It's' been tested via Deploy workflow runs as well as running `make install` locally

Reply:
That's misleading. Our direction is to use make targets but if we want to use helm direct then the version in there should be in sync with what's in the Makefile and that renders this PR useless.

Reply:
On second thought, I'll revert the changes done to the workflow (and docs etc.) regarding the repository/version update as well as NOT set it via Makefile. So the only change in this PR will be the `branch` info for every checkout in each of the workflow.